app/uk/gov/hmrc/digitalservicestax/frontend/Country.scala

```diff
@@ -44,7 +44,7 @@ object Country {
     )
     val raw = scala.io.Source.fromInputStream(stream).getLines.mkString
     val json = Json.parse(raw)
-    stream.close
+    stream.close()
```

Comment:
Interesting. I'm pretty sure that `Source.fromInputStream` in Scala closes the stream for us so this `close()` method call isn't necessary. You can open up the code for `Source` & follow the method calls & you will find this: `new BufferedSource(inputStream, bufferSize)(codec) withReset resetFn withClose close`. So it does close it.

To be honest I think the whole thing can be replaced with:

```
val source: String = Source.fromInputStream(getClass.getResourceAsStream("/public/location-autocomplete-canonical-list.json"))
        .getLines.mkString

Json.parse(source)
```

All of the existing vals make me think someone was doing a lot of debugging.

Reply:
looked into it and looks right, implemented it in the next commit